templates/typescript_gapic/_util.njk

```diff
@@ -231,14 +231,22 @@ limitations under the License.
  * @returns {Object}
  *   An iterable Object that allows [async iteration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols).
  *   When you iterate the returned iterable, each element will be an object representing
+ {%- if method.pagingMapResponseType %}
+ *   {{ typeLink(method.pagingMapResponseType) }}. The API will be called under the hood as needed, once per the page,
```

Comment:
This is not exactly correct: in this case each element will be an array of two elements, the first one is a string, and the second is pagingMapResponseType object, right?  The comment needs to be updated here.

Reply:
Agree. Updated.